docs/archive/windows/datagrid-guidance/customize-autogenerated-columns.md

```diff
@@ -0,0 +1,108 @@
+---
+title: How to - Customize Auto-Generated Columns in the DataGrid Control
+author: harinikmsft
+description: Guidance document that shows how to customize auto generated columns in the DataGrid control
+keywords: windows 10, uwp, windows community toolkit, windows toolkit, DataGrid, xaml control, xaml, AutoGenerateColumns
+---
+
+# How to: Customize Auto-Generated Columns in the DataGrid Control (Archived)
+
+**Important Note:** This document has been archived and the component is not available in the current version of the Windows Community Toolkit.
```

Comment:
For all these we should use the [DocFX important syntax](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#alerts):

```
> [!IMPORTANT]
> Whatever the note is, as is...
```